text/0055-sort.md

```diff
@@ -1,4 +1,4 @@
-- Title: Sort
+- Title: Sort //TODO CHECK IF IT NEEDS TO BE UPDATED OR THROW AWAY
```

Comment:
I've seen occurrences of old error codes. 

At this stage, I would like to know if we need to keep that specification because it's in the old format.

We added the search API and sortable-attributes API specification since it may suffice.

I will update the old error codes mentioned in that old spec for v1.0